tests/ui/apps/Scroller/ListOfThings/ScrollerListOfThings-View.js

```diff
@@ -0,0 +1,148 @@
+import Group from '@enact/ui/Group';
+import {Row, Column, Cell} from '@enact/ui/Layout';
+import ri from '@enact/ui/resolution';
+import spotlight from '@enact/spotlight';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import {Component, createRef} from 'react';
+
+import Button from '../../../../../Button/Button';
+import Dropdown from '../../../../../Dropdown/Dropdown';
+import Item from '../../../../../Item/Item';
+import Scroller from '../../../../../Scroller/Scroller';
+import ThemeDecorator from '../../../../../ThemeDecorator/ThemeDecorator';
+
+const ScrollerContainer = SpotlightContainerDecorator('div');
+const OptionsContainer = SpotlightContainerDecorator({leaveFor: {down: '#left'}}, 'div');
+const getScrollbarVisibility = (hidden) => hidden ? 'hidden' : 'visible';
+
+// NOTE: Forcing pointer mode off so we can be sure that regardless of webOS pointer mode the app
+// runs the same way
+spotlight.setPointerMode(false);
+
+const itemData = [];
+for (let i = 0; i < 100; i++) {
+	itemData.push({
+		children: `Item${i}`,
+		id: `item${i}`
+	});
```

Comment:
```suggestion
	itemData.push({
		children: `Item${i}`,
		id: `item${i}`,
		key: i
	});
```
Please add key to remove console error

Reply:
Suggested change.